Makefile

```
@@ -0,0 +1,6 @@
+NAME    := freeipa
+SRC_EXT := gz
+SOURCE  = https://releases.pagure.org/$(NAME)/$(NAME)-$(VERSION).tar.$(SRC_EXT)
+include packaging/Makefile_packaging.mk
+
+BUILD_DEFINES := --define "%snap +git10.87d5e59e6"
```

I'm not even sure it works this way, but usually you don't include the `%` in a `--define`:

```suggestion
BUILD_DEFINES := --define "snap +git10.87d5e59e6"
```

That said, I'm not sure this will work as is because:
https://github.com/daos-stack/freeipa/blob/2aa2b7c6bb416099514a55ce337252abbbdce0fb/packaging/Makefile_packaging.mk#L100
and
https://github.com/daos-stack/freeipa/blob/2aa2b7c6bb416099514a55ce337252abbbdce0fb/packaging/Makefile_packaging.mk#L108

so the `--define` will be used when getting the version.

You might have to define a new variable that is only used for the `rpmbuild` invocation:
https://github.com/daos-stack/freeipa/blob/2aa2b7c6bb416099514a55ce337252abbbdce0fb/packaging/Makefile_packaging.mk#L292-L293